rho/postprocessing.py

```diff
@@ -919,7 +920,11 @@ def escape_characters(data):
     """
     for key in data:
         if utilities.is_stringlike(data[key]):
-            data[key] = data[key].replace('\r\n', ' ').replace(',', ' ')
+            value = data[key].replace('\r\n', ' ').replace(',', ' ')
+            try:
+                data[key] = utilities.str_to_ascii(value)
+            except UnicodeEncodeError:
+                data[key] = ''
```

Comment:
Could we do `data[key] = value.encode('unicode_escape')`? I just tested on Python 2.7 and 3.6, and it seems to rewrite the string using `\uXXX` escapes for unicode characters.

Reply:
done